model/table_metadata.py

```diff
@@ -0,0 +1,30 @@
+from model.index import *
+
+class TableMetaData:
+    def __init__(self, table_schema, table_object):
+        self.__table_schema = table_schema
+        self.table_name = table_schema[SchemaKeys.NAME]
+        self.primary_key = table_schema[SchemaKeys.PRIMARY_KEY]
+        self.columns = table_schema[SchemaKeys.COLUMNS]
+        self.path = table_object.get_path()
```

Comment:
Why is the path here public while in the others private?

Reply:
Making them all one is a better idea.

Reply:
@YahyaAzzam  what do u mean by `Making them all one is a better idea.` ?

Reply:
+1 @MahmoudGoda2003